.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/" # Location of package manifests
```

Comment:
I'd recommend using
```
cooldown: 
  default-days: 7
```
As recommended by https://blog.yossarian.net/2025/11/21/We-should-all-be-using-dependency-cooldowns